js/modules/loki_app_dot_net_api.js

```diff
@@ -1221,6 +1228,17 @@ class LokiAppDotNetServerAPI {
 
     return this.uploadData(formData);
   }
+
+  putAvatar(buf) {
+    const formData = new FormData();
+    const buffer = Buffer.from(buf);
+    formData.append('avatar', buffer, {
+      contentType: 'application/octet-stream',
+      name: 'avatar',
+      filename: 'attachment',
+    });
+    return this.uploadAvatar(formData);
```

Comment:
should we remove this function from `web_api.js` then?

Reply:
I need to go and replace the avatar upload code before we can do this